lisa/tests/scheduler/load_tracking.py

```diff
@@ -902,6 +902,12 @@ class CPUMigrationBase(LoadTrackingBase):
     The duration of a single phase
     """
 
+    TASK_PERIOD_MS = 16
```

Comment:
Don't we already inherit that from `RTATestBundle` ?

Reply:
We do, but here that value is necessary for correctness. If we were to change the one from RTATestBundle, we really need this one to stay the same, they are not defined with the same constraints.